h2o-core/src/test/java/hex/AUCBuilderTest.java

```diff
@@ -0,0 +1,50 @@
+package hex;
+
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.zip.GZIPInputStream;
+
+import static org.junit.Assert.*;
+
+public class AUCBuilderTest {
+
+  @Test
+  public void testPerRow() throws Exception {
+    AUC2.AUCBuilder ab = new AUC2.AUCBuilder(400);
+    AUC2.AUCBuilder ab_old = new AUC2.AUCBuilder(400);
+    ab_old._speed = false;
+
+    long t = 0, t_old = 0;
+
+    try (GZIPInputStream gz = new GZIPInputStream(AUCBuilderTest.class.getResourceAsStream("aucbuilder.csv.gz"))) {
```

Comment:
Michal:  I assume the aucbuilder.csv.gz is the Higgs dataset?

Reply:
This is indeed based on the predictions from the Higgs dataset. It can, however, be generated random data as well.